.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -66,12 +66,10 @@ jobs:
 
       - name: Check query files (Unix)
         if: matrix.os != 'windows-2022'
-        env:
-          ALLOWED_INSTALLATION_FAILURES: haskell
         run: nvim --headless -c "luafile ./scripts/check-queries.lua" -c "q"
 
       - name: Check query files (Windows)
         if: matrix.os == 'windows-2022'
         env:
-          ALLOWED_INSTALLATION_FAILURES: haskell
+          ALLOWED_INSTALLATION_FAILURES: norg
```

Comment:
is this needed?

Reply:
https://github.com/nvim-treesitter/nvim-treesitter/runs/4386295723?check_suite_focus=true

Reply:
okay but that failure is different and not only `tree-sitter-norg`
```
Downloading...

Checking out locked revision

The system cannot find the path specified.

fatal: reference is not a tree: 74a721e8eed1a4a25cf495d45974ba24f315f81a

Error detected while processing command line:

Error while checking out revision

Failed to execute the following command:

{

  cmd = "git",

  err = "Error while checking out revision",

  info = "Checking out locked revision",

  opts = {

    args = { "checkout", "74a721e8eed1a4a25cf495d45974ba24f315f81a" },

    cwd = "C:\\Users\\runneradmin\\AppData\\Local\\nvim-data\\tree-sitter-prisma"

  }

}

Downloading...

Checking out locked revision

The system cannot find the path specified.

error: pathspec 'main' did not match any file(s) known to git

Error while checking out revision

Failed to execute the following command:

{

  cmd = "git",

  err = "Error while checking out revision",

  info = "Checking out locked revision",

  opts = {

    args = { "checkout", "main" },

    cwd = "C:\\Users\\runneradmin\\AppData\\Local\\nvim-data\\tree-sitter-norg"

  }

}

Downloading...

Checking out locked revision

The system cannot find the path specified.

fatal: reference is not a tree: 111b07762e86efab9a918b7c721f720c37e76b0a

Error while checking out revision

Failed to execute the following command:

{

  cmd = "git",

  err = "Error while checking out revision",

  info = "Checking out locked revision",

  opts = {

    args = { "checkout", "111b07762e86efab9a918b7c721f720c37e76b0a" },

    cwd = "C:\\Users\\runneradmin\\AppData\\Local\\nvim-data\\tree-sitter-typescript"

  }

}
```
and I think also 2-3 other failed with same reason that there was some issue with downloading

Reply:
Also the last run for gcc it was `graphql` and `html` having issues like this

Reply:
Master has same problem, maybe we should just fetch using curl on CI